src/agent/onefuzz-file-format/tests/files/binary-coverage.v1.json

```diff
@@ -1,15 +1,18 @@
 {
-    "/setup/main.exe": {
-        "blocks": {
-            "1": 0,
-            "12c": 1,
-            "1388": 0
-        }
-    },
-    "/setup/lib/some.dll": {
-        "blocks": {
-            "7b": 0,
-            "1c8": 10
+    "version": "1.0",
```

Comment:
We intentionally do _not_ add outer `version`/`coverage` keys to the `v0`/legacy test files, because we will only ever deserialize old files (which are always untagged).

In contrast, all non-legacy schemas will be deserialized and deserialized via the appropriate `..CoverageJson` enum, so the test files should be tagged.